### PR TITLE
Remove profile picture

### LINKED ISSUE #5243: Option to remove profile picture
Currently, there is no option to delete or remove a profile picture after it is uploaded. 
There must be one. 
Screenshot for the same:
Selection_086

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #5243 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

Thanks!

### CODE REVIEW

**Comment by divyabaid16** [comment]
Here I have the function as follows:
```  
  def profile_image
    if photo_file_name
      photo_path(:thumb)
    else
      "https://www.gravatar.com/avatar/#{OpenSSL::Digest::MD5.hexdigest(email)}"
    end
  end
```
Currently, the photo is removed from profile but it remains at navbar. 
I want to remove ```photo_file_name``` in the controller so that the photo can be removed from the navbar too. 

I would love any guidance here.
@jywarren @gauravano 
Thank you!

**Comment by SidharthBansal** [comment]
Awesome work. Can you please provide us some screenshots?

File: app/views/users/_photo.html.erb [hunk 1]
```diff
@@ -35,6 +35,15 @@
 </div>
 <p class="profile-uploading" style="display:none;padding-top:4px;color:#aaa;"><%= t('users._photo.uploading') %>...</p>
 
+<div style="font-size: 20px; text-align: center; margin-top: -20px;">
+  <% if @user.photo %>
```
Comment (gauravano) [inline app/views/users/_photo.html.erb]: I think, here check should be @user.photo_file_name as currently, option is visible to every user

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -321,6 +321,14 @@ def photo
     end
   end
 
+  def delete
+    @user = User.find(params[:id])
+    @user.photo = params[:photo]
+    @user[:photo_file_name] = ''
```
Comment (gauravano) [inline app/controllers/users_controller.rb]: It's not required as @user.photo.destroy will do the work

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -321,6 +321,14 @@ def photo
...
+    @user.photo.destroy
+    redirect_to "/profile/edit"
```
Comment (gauravano) [inline app/controllers/users_controller.rb]: Add @user.save! here

**Comment by divyabaid16** [comment]
GIF:
Peek 2019-04-15 22-37

**Comment by divyabaid16** [comment]
@gauravano I have made the changes.
Thanks!

**Comment by gauravano** [comment]
It's looking great @divyabaid16. How about writing a test for this? Thanks!

**Comment by divyabaid16** [comment]
@gauravano so should I write the test only for removing profile picture in test/functional/user_controller_test.rb ?

**Comment by gauravano** [comment]
Yes @divyabaid16!

**Comment by SidharthBansal** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**